config/default.yml

```diff
@@ -2951,6 +2951,11 @@ Style/DoubleNegation:
     - allowed_in_returns
     - forbidden
 
+Style/DuplicateRegexpCharacterClassElement:
```

Comment:
Isn't this supposed to be a Lint cop? :-)

Reply:
🤦